patches/minecraft/net/minecraft/entity/monster/EntityCreeper.java.patch

```diff
@@ -0,0 +1,50 @@
+--- ../src-base/minecraft/net/minecraft/entity/monster/EntityCreeper.java
++++ ../src-work/minecraft/net/minecraft/entity/monster/EntityCreeper.java
+@@ -1,5 +1,6 @@
+ package net.minecraft.entity.monster;
+ 
++import java.util.ArrayList;
+ import java.util.Collection;
+ import javax.annotation.Nullable;
+ import net.minecraft.entity.Entity;
+@@ -16,6 +17,7 @@
+ import net.minecraft.entity.ai.EntityAIWanderAvoidWater;
+ import net.minecraft.entity.ai.EntityAIWatchClosest;
+ import net.minecraft.entity.effect.EntityLightningBolt;
++import net.minecraft.entity.item.EntityItem;
+ import net.minecraft.entity.passive.EntityOcelot;
+ import net.minecraft.entity.player.EntityPlayer;
+ import net.minecraft.init.Items;
+@@ -184,22 +186,30 @@
+ 
+     public void func_70645_a(DamageSource p_70645_1_)
+     {
++    	if(net.minecraftforge.common.ForgeHooks.onLivingDeath(this, p_70645_1_)) return;
```

Comment:
@mezz I know that this will lead to duplicated events, but according to EntityPlayer (exact same behaviour already implemented) this shouldn't be a problem. Only other solution would be to 1 add a hasEventBeenCalled parameter to EntityLiving#onDeath() or b) make that method return a boolean value.